Sources/NIOTransportServices/StateManagedNWConnectionChannel.swift

```diff
@@ -259,19 +259,25 @@ extension StateManagedNWConnectionChannel {
             assert(self.pendingWrites.count == 0)
             return
         }
-
+        
         // Step 1 is to tell the network stack we're done.
         // TODO: Does this drop the connection fully, or can we keep receiving data? Must investigate.
         conn.cancel()
-
+        
         // Step 2 is to fail all outstanding writes.
         self.dropOutstandingWrites(error: error)
-
+        
```

Comment:
Can you remove the added whitespace here?

Reply:
It appears that on the main branch lines 262, 266, and 269 all have a single whitespace for those lines. Did I misunderstand what was needed?

Reply:
You added some spaces. :smile: